src/main/kotlin/no/skatteetaten/aurora/mokey/service/AuroraStatusCalculator.kt

```diff
@@ -8,25 +8,35 @@ import no.skatteetaten.aurora.mokey.model.PodDetails
 import no.skatteetaten.aurora.mokey.model.StatusCheck
 import no.skatteetaten.aurora.mokey.model.StatusCheckReport
 import no.skatteetaten.aurora.mokey.model.StatusCheckResult
-import no.skatteetaten.aurora.mokey.service.auroraStatus.DeploymentInProgressCheck
-import no.skatteetaten.aurora.mokey.service.auroraStatus.NoDeploymentCheck
-import no.skatteetaten.aurora.mokey.service.auroraStatus.OffCheck
+import no.skatteetaten.aurora.mokey.service.auroraStatus.AnyPodDownCheck
+import no.skatteetaten.aurora.mokey.service.auroraStatus.AnyPodObserveCheck
 import org.springframework.stereotype.Service
 import java.time.Instant
 import java.time.Instant.now
 
 @Service
-class AuroraStatusCalculator(val deploymentChecks: List<StatusCheck>) {
+class AuroraStatusCalculator(val statusChecks: List<StatusCheck>) {
 
     fun calculateAuroraStatus(app: DeployDetails, pods: List<PodDetails>, time: Instant = now()): AuroraStatus {
 
-        val results = (deploymentChecks).map {
+        val hasManagementHealthData = pods.any { it.managementData.health != null }
```

Comment:
Jeg synes denne testen bør utvides med it.managementData.health.isSuccess() == true